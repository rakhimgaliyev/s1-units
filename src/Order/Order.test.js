import React from 'react';
import Order from './Order';
import {shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import { wrap } from 'module';

configure({ adapter: new Adapter() });

describe('Order component', () => {
  it('only date and shop here', () => {
    const order = {
      shop: 'Adidas',
      date: '939686400'
    }
    const wrapper = shallow(<Order order={order}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('only date and shop here', () => {
    let order = {
      shop: 'Adidas',
      date: '939686400',
      items: [{
        snippets: 1,
      }]
    }
    const wrapper = shallow(<Order order={order}/>);
    order.items = [
      {
        snippets: 1
      },
      {
        socks: 2
      },
    ];
    wrapper.setProps({
      order: order 
    })
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
