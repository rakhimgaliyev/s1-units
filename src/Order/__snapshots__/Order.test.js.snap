// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order component only date and shop here 1`] = `
<div
  className="Order"
>
  <div
    className="Order-header"
  >
    <span
      className="Order-shop"
    >
      Adidas
    </span>
    <span>
      NaN undefined, undefined, NaN год
    </span>
  </div>
  <div
    className="Order-items"
  />
</div>
`;

exports[`Order component only date and shop here 2`] = `
<div
  className="Order"
>
  <div
    className="Order-header"
  >
    <span
      className="Order-shop"
    >
      Adidas
    </span>
    <span>
      NaN undefined, undefined, NaN год
    </span>
  </div>
  <div
    className="Order-items"
  >
    <div
      className="Order-item"
      key="0"
    >
      <Component />
    </div>
  </div>
</div>
`;
